### DISCUSSION TITLE
There's a installation problem regarding .NET from Microsoft.

### DISCUSSION THREAD

**Discussion by TeddyTheGamer405** [shaked6540/YoutubePlaylistDownloader#170] [General]
So I reinstalled Windows recently and I was trying to reinstall this, It requires me to download .NET from Microsoft's website, However when I went to install the plugin required for the app to work it still doesn't work. Idk what happened it was working fine before.

**Comment by oxpsyxo** [comment]
Downloading the SDK : 
.NET 7.0 SDK (v7.0.306) - Windows x64 Installer
Instead of the link in the prompt got it up for me.
https://dotnet.microsoft.com/en-us/download/dotnet/thank-you/sdk-7.0.306-windows-x64-installer

I got the x64 version but you can use the one relevant to you

**Reply by JJs-Code** [reply]
thanks, this worked for me too
